seafile/SeafAppDelegate.m

```diff
@@ -103,10 +103,11 @@ - (void)enterAccount:(SeafConnection *)conn
         }
     }
     if (updated)
+        [SeafDataTaskManager.sharedObject startLastTimeUnfinshTaskWithConnection:conn];
         [self.tabbarController setSelectedIndex:TABBED_SEAFILE];
```

Comment:
Indentation here seems to indicate that `[self.tabbarController setSelectedIndex:TABBED_SEAFILE];` will execute only `if (updated)` but there's no brace so it will always execute. The spacing is confusing for what was intended.